media-video/obs-studio/obs-studio-21.1.2.ebuild

```diff
@@ -6,7 +6,7 @@ EAPI=6
 PYTHON_COMPAT=( python{3_4,3_5,3_6} )
 CMAKE_MIN_VERSION=3.9.6
```

Comment:
`3.9.6` is current default in `cmake-utils.eclass` so this line could be dropped

Reply:
Dropped it shall be.